ext/pdo/pdo_dbh.c

```diff
@@ -341,6 +341,7 @@ static PHP_METHOD(PDO, dbh_constructor)
 	}
 
 	dbh->auto_commit = pdo_attr_lval(options, PDO_ATTR_AUTOCOMMIT, 1);
+    dbh->error_mode = pdo_attr_lval(options, PDO_ATTR_ERRMODE, PDO_ERRMODE_EXCEPTION);
```

Comment:
Should use tabs.

Reply:
Oops. Fixed. Thanks - I'm using a different editor and "show tabs" wasn't enabled like I expect it to be, so I just assumed it was all spaces.